client/landing/gutenboarding/components/plans/plans-button/index.tsx

```diff
@@ -38,8 +38,10 @@ const PlansButton: React.FunctionComponent< Button.ButtonProps > = ( { ...button
 
 	const plan = useSelectedPlan();
 
-	/* translators: Button label where %s is the WordPress.com plan name (eg: Free, Personal, Premium, Business) */
-	const planLabel = sprintf( __( '%s Plan' ), plan.getTitle() );
+	const planLabel = ! plan?.isFree
+		? /* translators: Button label where %s is the WordPress.com plan name (eg: Personal, Premium, Business) */
+		  sprintf( __( '%s Plan' ), plan.getTitle() )
+		: __( 'View plans' );
```

Comment:
I think we have some "interacted with plans" selector that we could add here, so that we show "free plan" if they really actually selected it from the grid.

Reply:
Do we want to show the "Free Plan" text if they've actually selected free or just leave the "View plans" text (as the PR currently does)?

There's also some odd behavior where selecting a domain implies a paid plan, but you can select the free plan which leaves things in an odd state…

@johnHackworth

Reply:
In case there's no clarity, I think it's fine to merge-as is.

Reply:
If the user has selected the free plan, let's show "free plan" there

Reply:
(if the user has selected a domain, let's keep whatever we were putting there (premium?), too)